### ISSUE TITLE
Grouping messages by "group" segments

### ISSUE THREAD

**Issue by Chemaclass** [Chemaclass/EdifactParser#7]
# Current behavior

Currently, only the "message segment grouping" it's supported. This is done by grouping the messages around a `TransactionMessage` object.

But, as you can see in the [Service Segments](https://github.com/Chemaclass/EdifactParser/blob/master/docu/what-is-edifact.md), we could have an extra group definition within the "Interchange" segments.

```
UNB - Start of Interchange
UNG - Start of Group <- this is completely missing right now
UNH - Start of Message
UNT - End of Message
UNE - End of Group <- this is completely missing right now
UNZ - End of Interchange
```

The actual grouping (by message segment level) it's done in `TransactionMessage::groupSegmentsByMessage()`.

# Acceptance Criteria

* https://github.com/Chemaclass/EdifactParser/issues/6 needs to be done first in order to be able to group the message segments properly (by the ending message segment definition).
* Group the list of messages in a list of groups surrounding the UNG and UNE segments.
* Ignore every line that it's not between a UNG(start of a group) and UNE (end of a group) segments.

A test would be:
```
UNG+1' <- it starts the first group 
IGN+ORE+ME' <- This line should be ignored and not included in any message
UNH+1+anything'  <- it starts the first message
UNT+19+1'  <- it finishes the first message
UNE+1+2' <- it finishes the first group 

IGN+ORE+ME' <- This line should be ignored and not included in any group

UNG+2'  <- it starts the first group 
UNH+2+anything'  <- it starts the second message
UNT+19+2' <- it finishes the second message
UNE+2+2' ' <- it finishes the first group 

UNZ+2+3' <- it ends of Interchange
```
[Note: the new lines in the previous example are just to help the visualization of the separation of each group and what represent each line]

**Comment by Chemaclass** [comment]
Already done by @JesusValera
